### PR TITLE
update swagger-ui-dist to 5.6.2

### PR DESCRIPTION
**Pull request by dambaev**
Had to modify swagger-ui.css in order to satisfy postcss loader

### CODE REVIEW

**Comment by dambaev** [comment]
so tldr: the latest swagger-ui version, which runs without issues now (ie: integrated with our frontend) is **5.6.2**. Disabling/enabling build optimization has **no** effect here. All newer versions compile with errors we saw initially and if the there will be a requirement to update swagger ui then it should be splitted out from frontend and be deployed as a separate app in order to not to interfere with angular's build optimizations, webpacks and so on.

This version still does not show error body, but have proper mobile layout. @Natsvora discovered, that swagger don't show error body due to 4xx error response has no content-type defined. Thus there is https://github.com/op-energy-foundation/op-energy/pull/13 PR to make errors to return responses in JSON format to satisfy swagger-ui component. It had been confirmed that error response in JSON format allowes swagger-ui to display it's body

**Review by Natsvora — APPROVED** [review]
